models/app.model.js

```diff
@@ -7,16 +7,34 @@ const fetchCategories = (request, response) => {
     })
 }
 
-const fetchReviews = () => {
-  
-    const reviewCommentCount = 
+const fetchReviews = (category, sort_by="created_at", order="desc") => {
+    const acceptedCategories = ["euro game", "social deduction", "dexterity", "children's games"];
+
+    const acceptedSortBys = ["review_id","title", "designer", "owner", "review_img_url", "review_body", "category", "created_at", "votes"]
```

Comment:
This, however, is okay to be hard coded like it is. All reviews will always follow this same format - as we have dictated in our seeding.